python/lsst/ts/ATMCSSimulator/mcs_csc.py

```diff
@@ -584,9 +580,14 @@ def set_event(self, evt_name, **kwargs):
             Event name (without the ``evt_`` prefix)
         **kwargs : `dict`
             Data for ``ControllerEvent.set``
+
+        Returns
+        -------
+        did_put : `bool`
```

Comment:
Please consider removing the variable and just declaring a type. The type only specification is what the NumPy docstrings documentation recommends.

Reply:
I withdraw the above comment. It's OK as is.